### ISSUE TITLE
[feature]: Skip create default workspace automatically

### ISSUE THREAD

**Issue by KarloDerEchte** [makeplane/plane#2432]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

The feature request involves implementing an option to allow new users to automatically skip the "create workspace or skip" step upon registration, eliminating the need for this manual step.

### Why should this be worked on?

This feature would enhance the user onboarding experience by streamlining the registration process. It's particularly beneficial for organizations or platforms where workspace creation is not a mandatory step for all users. By automating the skip process, new users can access the platform more quickly and efficiently, improving their overall experience and saving time during onboarding.

**Comment by contigen** [comment]
Hi @maintaners. Regarding this issue, I tried the app, the onboarding seems OK to me. But, if you consider this an issue, I'd like to work on it. Thanks!

**Comment by jmessy919** [comment]
Yeah, I agree @KarloDerEchte - I tried demoing this out with a colleague of mine and in order for him to get through the onboarding process, he had to create a personal workspace, get to the dashboard, and delete the personal workspace he just created. And this was a user I directly invited to an existing workspace. Making this process non-mandatory for installations that already have at least 1 workspace to which the new user user has been invited would drastically reduce onboarding friction.

**Comment by vihar** [comment]
This works now — users arriving with a pending invite skip the workspace creation step and go straight through onboarding, so there's no personal workspace to create and delete afterwards.

Closing this out.
